pkg/v1/validate/index.go

```diff
@@ -75,7 +76,7 @@ func validateChildren(idx v1.ImageIndex) error {
 				errs = append(errs, fmt.Sprintf("failed to validate image MediaType[%d](%s): %v", i, desc.Digest, err))
 			}
 		default:
-			return fmt.Errorf("todo: validate index Blob()")
+			logs.Warn.Printf("Unexpected manifest: %s", desc.MediaType)
```

Comment:
Do we want to return here?

Reply:
We need to add a `Blob` method to `v1.ImageIndex` if I want to fix that. I can send a PR for that as well, but as it is this will break our tests if I return an error. I'm pretty sure I *do* want to add `Blob` to that interface, but I'd like someone to tell me it's a good idea first :)